nix/packages/default.nix

```diff
@@ -2,14 +2,33 @@
   perSystem = { pkgs, inputs', system, config, ... }:
     {
       packages = {
+        refresh-available-snapshots-state = pkgs.writeShellApplication {
+          name = "refresh-available-snapshots-state";
+          runtimeInputs = with pkgs; [ awscli2 jq ];
+          text =
+            let
+              # TODO: handle this variable better (e.g. move in a better place, some vars are duplicated in scheduled tasks) 
+              awsEndpoint = "https://pub-b887f41ffaa944ebaae543199d43421c.r2.dev/";
+              outFile = "available-snapshots.json";
+              bucketName = "cardanow";
```

Comment:
Don't you have access to variables from `setup_env_vars.sh` here?
Otherwise you could simply do something like `source ${../../setup_env_vars.sh}` here? Just a very spontaneous idea, have no idea it really makes sense.

Reply:
That's a good idea but `setup_env_vars.sh` depends on some `NETWORK` var but _this_ command is network independent. I think I'd prefer to think more carefully how we handle env vars in the near future and leave this